src/main/java/Calendar1/Cal1.java

```diff
@@ -0,0 +1,11 @@
+package Calendar1;
+
+public class Cal1 {
```

Comment:
No idea what are you calculating

Reply:
I see, it's the advent calculator. You really sohuld have attached the task description with the PR